client/components/forms/form-radio-with-thumbnail/index.jsx

```diff
@@ -0,0 +1,44 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import FormRadio from 'components/forms/form-radio';
+import FormLabel from 'components/forms/form-label';
+import cssSafeUrl from 'lib/css-safe-url';
+
+const FormRadioWithThumbnail = ( { label, thumbnail, ...otherProps } ) => {
+	const { cssClass, cssColor, imageUrl } = thumbnail;
+	const styles = {
+		backgroundColor: cssColor,
+		backgroundImage: imageUrl ? 'url(' + cssSafeUrl( imageUrl ) + ')' : '',
+	};
```

Comment:
Seems semantically inappropriate to use CSS and especially `style` to do something like this.

Why is this using CSS instead of just adding an `<img>`?